ImperatorToCK3/CK3/Titles/LandedTitles.cs

```diff
@@ -464,5 +464,50 @@ ICollection<Governorship> countryLevelGovernorships
 
 			return countyHoldersCache;
 		}
+
+		public void ImportDevelopmentFromImperator(Imperator.Provinces.ProvinceCollection imperatorProvinces, ProvinceMapper provMapper, Date date) {
+			Logger.Info("Importing development from Imperator...");
+
+			var counties = this.Where(t => t.Rank == TitleRank.county);
+			var impProvsPerCounty = new Dictionary<string, int>();
+
+			var ck3ProvsPerImperatorProv = new Dictionary<ulong, int>(); // Imperator province ID, number of CK3 provs mapped to it
+			foreach (var county in counties) {
+				var imperatorProvs = new HashSet<ulong>();
+				foreach (var ck3ProvId in county.CountyProvinces) {
+					foreach (var impProvId in provMapper.GetImperatorProvinceNumbers(ck3ProvId)) {
+						imperatorProvs.Add(impProvId);
+						ck3ProvsPerImperatorProv.TryGetValue(impProvId, out var currentValue);
+						ck3ProvsPerImperatorProv[impProvId] = currentValue + 1;
+					}
+				}
+
+				impProvsPerCounty[county.Id] = imperatorProvs.Count;
+			}
+
+			foreach (var county in counties) {
+				if (impProvsPerCounty[county.Id] == 0) {
```

Comment:
You could make a small function called IsCountyOutsideImperatorMap that does this conditional check. It would make the code in the loop more expressive and eliminate the need for the comment.